uiowa/uiowa
# [ITS] Support article content type

Resolves https://github.com/uiowa/uiowa/issues/7414.

# How to test
```
ddev blt frontend && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /node/add/support_article
```
- Create an article and set as draft, confirm moderation block appears
- Confirm fields listed on https://github.com/uiowa/uiowa/issues/7414
- Confirm content type uses ([our best practices](https://iowa.sharepoint.com/sites/drupal-developer-documentation/SitePages/Miscellaneous/Content%20type%20creation%20notes/Content%20type%20creation%20notes.aspx) )
- Confirm url uses node id
- Confirm FAQs accordion aria is wired up correctly
- Confirm mockup looks like https://sandbox.prod.drupal.uiowa.edu/phishing-examples
